### ISSUE TITLE
fetching any JSON file that is not `graph.json` always returns an ES module

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#604]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
Just noticed as part of working on #516 that if you try and do something like this
```html
<script type="module">
  fetch('/data.json')
    .then(resp => resp.json())
    .then(resp => console.debug('DATA', resp));
</script>
```

Based on [our current logic](https://github.com/ProjectEvergreen/greenwood/blob/v0.12.0/packages/cli/src/plugins/resource/plugin-standard-json.js#L26), it will always come back as ESM, e.g.
```json
export default { ... }
```

When it should just come back as the exact contents.

## Details
This also uncovers another issue related to how we should be able to support using ESM with non standard module types like JSON and images, since `import` just uses `fetch`, to a browser, a request for JSON, to a browser they both look the same, so there is currently no way for us to distinguish the differences between these two
```html
<script type="module">
  fetch('/data.json')
    .then(resp => resp.json())
    .then(resp => console.debug('DATA', resp));
</script>
```

```js
import json from '/package.json';
```

Will need to setup a discussion for this and drop down to using an internal API for now that can get "hints" along the lines of import assertions to help keep us moving, ex.
```js
import json from '/package.json?type=json';
```

> ~~**_However, this need for query params is dependent on #599 and requires a bit of a work around for now, so needing to make an exception to solve both at the same time._**~~
